tokio-trace/Cargo.toml

```diff
@@ -8,7 +8,7 @@ name = "tokio-trace"
 #   - README.md
 # - Update CHANGELOG.md.
 # - Create "v0.1.x" git tag
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Why do you deprecate and bump the version?

Reply:
I see it is deprecated in trace-core

Reply:
Ah, yeah, I think this was a mistake --- I'll un-bump it.